src/app/components/Footer.jsx

```diff
@@ -0,0 +1,92 @@
+import SocialMediaButtons from "../components/common/SocialMediaBtn";
```

Comment:
also could you please rename that folder to SocialMediaButton? I didn't notice that you've decided to shorten it